test/evergreen.yml

```diff
@@ -4321,6 +4321,34 @@ buildvariants:
     - name: cppsuite-search-near-02-stress
     - name: cppsuite-search-near-03-stress
 
+- name: cppsuite-stress-tests-arm64
+  display_name: "Cppsuite Stress Tests ARM64"
+  batchtime: 720 # twice a day
+  run_on:
+  - ubuntu2004-arm64-large
+  expansions:
+    test_env_vars:
+      WT_TOPDIR=$(git rev-parse --show-toplevel)
+      WT_BUILDDIR=$WT_TOPDIR/cmake_build
+      LD_LIBRARY_PATH=$WT_BUILDDIR:$WT_TOPDIR/TCMALLOC_LIB/lib
+      LD_PRELOAD=$WT_TOPDIR/TCMALLOC_LIB/lib/libtcmalloc.so
+    cmake_generator: Ninja
+    make_command: ninja
+    CMAKE_PREFIX_PATH: -DCMAKE_PREFIX_PATH="$(pwd)/TCMALLOC_LIB"
+  tasks:
+    - name: compile
+    - name: cppsuite-operations-test-stress
+    - name: cppsuite-burst-inserts-stress
+    - name: cppsuite-bounded-cursor-prefix-indices-stress
+    - name: cppsuite-bounded-cursor-prefix-search-near-stress
+    - name: cppsuite-bounded-cursor-prefix-stat-stress
+    - name: cppsuite-bounded-cursor-stress-reverse-stress
+    - name: cppsuite-bounded-cursor-stress-stress
+    - name: cppsuite-hs-cleanup-stress
+    - name: cppsuite-search-near-01-stress
+    - name: cppsuite-search-near-02-stress
+    - name: cppsuite-search-near-03-stress
```

Comment:
Suggest using [task tag & selector](https://github.com/evergreen-ci/evergreen/wiki/Project-Configuration-Files#task-tags) for this set of CPP stress tasks, for code simplification and reusability.